basic-ruby/spec/filters_dinosaurs_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
```

Comment:
If you're not going to add tests (you probably should wait for the next chapter of levelup) remove this and all the other test files.